exporters/metrics/ocagent/src/main/java/io/opencensus/exporter/metrics/ocagent/OcAgentMetricsExporter.java

```diff
@@ -72,6 +73,37 @@ public static void createAndRegister(OcAgentMetricsExporterConfiguration configu
         configuration.getRetryInterval());
   }
 
+  /**
+   * Enables OpenCensus metric.
+   *
+   * <p>This will register all basic {@link io.opencensus.stats.View}s. When coupled with an agent,
+   * it allows users to monitor application behavior.
+   *
+   * <p>Example usage for maven:
+   *
+   * <pre>{@code
+   * <dependency>
+   *   <groupId>io.opencensus</groupId>
+   *   <artifactId>opencensus-exporter-metrics-ocagent</artifactId>
+   *   <version>${opencensus.version}</version>
+   * </dependency>
+   * }</pre>
+   *
+   * <p>It is recommended to call this method before doing any RPC call to avoid missing stats.
+   *
+   * <pre>{@code
+   * OcAgentMetricsExporter.basicSetup(OcAgentMetricsExporterConfiguration.builder().build());
+   * }</pre>
+   *
+   * @param configuration the {@code OcAgentMetricsExporterConfiguration}.
+   * @since 0.25
+   */
+  public static void basicSetup(OcAgentMetricsExporterConfiguration configuration) {
+    // register basic rpc views
+    RpcViews.registerAllGrpcBasicViews();
```

Comment:
what about http views? 

Maybe the description should also say that basicSetup also creates and registers the exporter so createAndRegister() api is not required.